ui/pages/remote-mode/setup/setup-daily-allowance/remote-mode-setup-daily-allowance.component.tsx

```diff
@@ -169,27 +158,39 @@ export default function RemoteModeSetupDailyAllowance() {
       return;
     }
 
-    const newAllowance = {
-      tokenType: selectedAllowanceToken,
+    const selectedAsset = assetsWithBalance.find(
```

Comment:
Currently getting all the testnet tokens showing up (including `SOL`). Also I'm not able to select a different token or add anything (specifically clicking add doesn't do anything). Not sure if it's just me?

![image](https://github.com/user-attachments/assets/b42c4b2b-402b-4f0f-8286-8aef55a85f08)